packages/node/src/utils/hot-reload.ts

```diff
@@ -168,7 +168,7 @@ export const performReload = async (
     }
   });
   //@ts-ignore
-  __webpack_require__.federation.instance.moduleCache.clear();
+  __webpack_require__?.federation?.instance?.moduleCache?.clear();
```

Comment:
NOTE to reviewers: The code above this already clears the module cache of all the registered remotes. This line assumes that the current execution context is within a federated remote built using federation + webpack, which might not be the case